### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by nakengelhardt**

### CODE REVIEW

File: docs/source/mutate.rst [hunk 1]
```diff
@@ -1,4 +1,89 @@
 Mutation export options
 =======================
 
-TBD
+The first step in any test script is to obtain the mutated module to run the test on.
+
+There are three configurations that are commonly used:
+
+1. Permanently enabled single mutation. Applying a single mutation without any control options results in a module with he same interface as the original module that can be substituted in a testbench without requiring any modifications. This is generally the most straightforward way to run the testsuite that is being evaluated.
```
Comment (mattvenn) [inline docs/source/mutate.rst:8]: with the same

File: docs/source/config.rst [hunk 2]
```diff
@@ -4,42 +4,48 @@ Configuration File Format
...
 Mutation generation options: for a description of how these values influence the mutation generation algorithm, see :ref:`mutgen`.
 
-weight_cover
-	Default: 500
+``weight_cover``
+	Optional. Default: 500
 
-weight_pq_w weight_pq_b weight_pq_c weight_pq_s weight_pq_mw weight_pq_mb weight_pq_mc weight_pq_ms
-	Default: 100
+``weight_pq_w weight_pq_b weight_pq_c weight_pq_s weight_pq_mw weight_pq_mb weight_pq_mc weight_pq_ms``
```
Comment (mattvenn) [inline docs/source/config.rst:30]: worth putting a link here to the description of what these do (in your other commit)

Comment (nakengelhardt) [reply docs/source/config.rst:30]: A different link than the one just above on line 25?

Comment (mattvenn) [reply docs/source/config.rst:30]: I think the same link, just repeated. Those terms are all new to me and it wasn't obvious the link in the previous section would also show me what these terms mean.

**Pull request merged**